DevTools/HackStudio/CursorTool.cpp

```diff
@@ -147,6 +147,8 @@ void CursorTool::on_keydown(GUI::KeyEvent& event)
         case Key_Right:
             move_selected_widgets_by(m_editor.form_widget().grid_size(), 0);
             break;
+        default:
+            break;
```

Comment:
I don't get your persistence on this, its existence makes no difference, and this exact behaviour is implied by its nonexistence.

Not nitpicking, just wondering out loud

Reply:
😀 it's not me, blame GCC

Reply:
Oh there are more enum values, I see.
\*blames GCC\*